org.eclipse.jdt.ui.tests/ui/org/eclipse/jdt/ui/tests/quickfix/CleanUpTest.java

```diff
@@ -17450,29 +17452,28 @@ public void testOverriddenAssignment() throws Exception {
 				+ "public class E {\n" //
 				+ "    public boolean removeUselessInitialization() {\n" //
 				+ "        // Keep this comment\n" //
-				+ "        boolean reassignedVar;\n" //
-				+ "        reassignedVar = \"\\n\".equals(File.pathSeparator);\n" //
+				+ "        boolean reassignedVar = \"\\n\".equals(File.pathSeparator);\n" //
 				+ "        return reassignedVar;\n" //
 				+ "    }\n" //
 				+ "\n" //
 				+ "    public long removeInitForLong() {\n" //
 				+ "        // Keep this comment\n" //
 				+ "        long reassignedVar;\n" //
+				+ "        System.out.println();\n" //
 				+ "        reassignedVar = System.currentTimeMillis();\n" //
 				+ "        return reassignedVar;\n" //
 				+ "    }\n" //
 				+ "\n" //
 				+ "    public String removeInitForString() {\n" //
 				+ "        // Keep this comment\n" //
-				+ "        String reassignedVar;\n" //
-				+ "        reassignedVar = File.pathSeparator;\n" //
+				+ "        String reassignedVar = File.pathSeparator;\n" //
```

Comment:
Similarly, I would expect the code to end up:

System.out.println();
String reassignedVar = File.pathSeparator;